register receivers as not exported when neither export flag is set

Smali changes of vendor_oplus_camera c00ca91, over baksmali --sequential-labels output.

diff --git a/classes2/com/oplus/gallery/foundation/uikit/broadcast/bus/ActionReceiver.smali b/classes2/com/oplus/gallery/foundation/uikit/broadcast/bus/ActionReceiver.smali
index 2309c1361469015752f154b4711fa822775508f0..37e01b4b42ad229b59f517f7f4836756f8ccdedc 100644
--- a/classes2/com/oplus/gallery/foundation/uikit/broadcast/bus/ActionReceiver.smali
+++ b/classes2/com/oplus/gallery/foundation/uikit/broadcast/bus/ActionReceiver.smali
@@ -189,19 +189,26 @@
     .line 61
     iget v7, v0, Lcom/oplus/aiunit/vision/lu0;->c:I
 
+    and-int/lit8 v1, v7, 0x6
+
+    if-nez v1, :cond_44
+
+    or-int/lit8 v7, v7, 0x4
+
+    :cond_44
     move-object v3, p0
 
     .line 64
     invoke-virtual/range {v2 .. v7}, Landroid/content/Context;->registerReceiver(Landroid/content/BroadcastReceiver;Landroid/content/IntentFilter;Ljava/lang/String;Landroid/os/Handler;I)Landroid/content/Intent;
-    :try_end_42
-    .catchall {:try_start_b .. :try_end_42} :catchall_46
+    :try_end_48
+    .catchall {:try_start_b .. :try_end_48} :catchall_4c
 
     .line 67
     invoke-static {}, Lcom/oplus/aiunit/vision/whg;->f()V
 
     return-void
 
-    :catchall_46
+    :catchall_4c
     move-exception v0
 
     move-object p0, v0
